### ISSUE TITLE
empty_count still throwing false positives

### ISSUE THREAD

**Issue by gwsounddsg** [realm/SwiftLint#1657]
issue #1622 closed this exact issue and was updated in v0.20 but false positives are still happening. Here's an example:

```swift
mutating func set(outputCount count: Int) throws {
        try check(count)
        if count == 0 { // empty_count error
                throw AveModelErr.MixerStack_badOutput(count)
        }
}
```

**Comment by marcelofabri** [comment]
This is not the same issue as #1622, it's a duplicate of #827.

**Comment by gwsounddsg** [comment]
@marcelofabri could you elaborate on how this is not the same as #1622? In the example code I showed swiftlint is throwing an `empty_count` error.

**Comment by marcelofabri** [comment]
#1622 is about words ending with `count`, such as `discount`. Please take a look at #827 for more discussion about your case.
